diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/StringTemplateFormatRule.kt

```diff
@@ -77,4 +85,17 @@ class StringTemplateFormatRule(private val configRules: List<RulesConfig>) : Rul
             }
         }
     }
+
+    @Suppress("UnsafeCallOnNullableType")
+    private fun bracesCanBeOmitted(node: ASTNode): Boolean {
+        return if (node.findAllNodesWithSpecificType(REFERENCE_EXPRESSION).size == 1) {
+            node.findAllNodesWithSpecificType(REFERENCE_EXPRESSION).first().treeParent.elementType == LONG_STRING_TEMPLATE_ENTRY
```

Comment:
You can call `singleOrNull()` on a list instead of checking size and then accessing `first`.